app/src/main/res/layout/fragment_credit_card_editor.xml

```diff
@@ -2,198 +2,204 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:layout_margin="16dp"
-    android:orientation="vertical">
-
-    <!-- Card Number -->
-    <TextView
-        android:id="@+id/card_number_title"
-        android:layout_width="wrap_content"
-        android:layout_height="16dp"
-        android:gravity="center_vertical"
-        android:paddingStart="3dp"
-        android:paddingEnd="0dp"
-        android:text="@string/credit_cards_card_number"
-        android:textColor="?primaryText"
-        android:textSize="12sp"
-        android:letterSpacing="0.05"
-        app:fontFamily="@font/metropolis_semibold" />
-
-    <com.google.android.material.textfield.TextInputLayout
-        android:id="@+id/card_number_layout"
+    android:layout_height="match_parent">
+
+    <LinearLayout
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:textColor="?primaryText"
-        app:errorEnabled="true"
-        app:hintEnabled="false">
+        android:layout_margin="16dp"
+        android:orientation="vertical"
+        android:paddingBottom="80dp">
```

Comment:
Why did we add this paddingBottom?

Reply:
Because otherwise the buttons would hit the very bottom. It doesn't have to be 80 but it didn't seem to affect anything either way. Just need some padding between the buttons and the edge of the screen.